pkg/envoy/server.go

```diff
@@ -1011,7 +1011,6 @@ func createBootstrap(filePath string, nodeId, cluster string, xdsSock, egressClu
 			},
 		},
 		Admin: &envoy_config_bootstrap.Admin{
-			AccessLogPath: "/dev/null",
```

Comment:
Was this intentional?

Reply:
Yes, sorry for not pointing this out in the commit message. `AccessLogPath` has been deprecated. Removing it eliminates a deprecation warning from Cilium agent logs.